ext/standard/config.w32

```diff
@@ -28,14 +28,15 @@ EXTENSION("standard", "array.c base64.c basic_functions.c browscap.c \
 	crypt_sha512.c  php_crypt_r.c crc32_x86.c \
 	datetime.c dir.c dl.c dns.c dns_win32.c exec.c \
 	file.c filestat.c formatted_print.c fsock.c head.c html.c image.c \
-	info.c iptc.c lcg.c link.c mail.c math.c md5.c metaphone.c microtime.c \
-	net.c pack.c pageinfo.c quot_print.c rand.c mt_rand.c soundex.c \
-	string.c scanf.c syslog.c type.c uniqid.c url.c var.c \
-	versioning.c assert.c strnatcmp.c levenshtein.c incomplete_class.c \
-	url_scanner_ex.c ftp_fopen_wrapper.c http_fopen_wrapper.c \
-	php_fopen_wrapper.c credits.c css.c var_unserializer.c ftok.c sha1.c \
-	user_filters.c uuencode.c filters.c proc_open.c password.c \
-	streamsfuncs.c http.c flock_compat.c random.c hrtime.c", false /* never shared */,
+	image_avif/avifinfo.c info.c iptc.c lcg.c link.c mail.c math.c \
```

Comment:
Adding `image_avif/avifinfo.c` here doesn't work on Windows. Instead add a `ADD_SOURCES` "macro" after the `EXTENSION` "macro":
````.js
ADD_SOURCES("ext/standard/image_avif", "avifinfo.c", "standard");
````